berserk/clients/bulk_pairings.py

```diff
@@ -0,0 +1,91 @@
+"""Bulk pairings client"""
+from __future__ import annotations
+
+from typing import Any, cast, Dict, List, Optional
+
+from .base import BaseClient
+from ..enums import Variant
+from ..formats import JSON_LIST, JSON
+from ..types.bulk_pairings import BulkPairingGame
+from ..types.core.aliases import ID
+
+
+class BulkPairings(BaseClient):
+    """Client for bluk pairing related endpoints."""
+
+    def view_upcoming(self) -> List[BulkPairingGame]:
+        """View upcoming bulk pairings.
+
+        :return: List of upcoming bulk pairings.
+        :rtype: List[BulkPairingGame]
+        """
+        path: str = "/api/bulk-pairing"
+        return cast(List[BulkPairingGame], self._r.get(path, fmt=JSON_LIST))
+
+    def create(
+        self,
+        players_tokens: List[str],
+        clock_limit: int,
+        clock_increment: int,
+        days: Optional[int] = None,
+        pair_at: Optional[int] = None,
+        start_clocks_at: Optional[int] = None,
+        rated: Optional[bool] = None,
+        variant: Optional[Variant] = None,
+        fen: Optional[str] = None,
+        message: Optional[str] = None,
+        rules: Optional[list[str]] = None,
+    ) -> BulkPairingGame:
+        """Create a bulk pairing.
+
+        :param List[str] players_tokens: players OAuth tokens
+        :param int clock_limit: clock initial time
+        :param int clock_increment: clock increment
+        :param Optional[int] days: days per turn (correspondence)
+        :param Optional[int] pair_at: day at wich game will be created
+        :param Optional[int] start_clocks_at: day at wich clocks will start
+        :param Optional[bool] rated: rated or casual
+        :param Optional[Variant] variant: game variant
+        :param Optional[str] fen: starting fen
+        :param Optional[str] message: message sent to players
+        :param Optional[list[str]] rules: extra game rules
+        :return BulkPairingGame: the pairing created
+        """
+        path: str = "/api/bulk-pairing"
+        payload: Dict[str, Any] = {
```

Comment:
the explicit typing here is unnecessary

Reply:
Oh, it's just my own code style, will fix.